pkg/verifier/aws/entry_point.go

```diff
@@ -206,6 +207,43 @@ func (a *AwsVerifier) ValidateEgress(vei verifier.ValidateEgressInput) *output.O
 
 	// Terminate the EC2 instance (unless user requests otherwise)
 	if !vei.SkipInstanceTermination {
+
+		//Replaced the SGs atached to the network-verifier-instance by the default SG in order to allow
+		//deletion of temporary SGs created
+
+		//Getting a list of the SGs for the current VPC of our instance
+		var defaultSecurityGroupID = ""
+		describeSGOutput, err := a.AwsClient.DescribeSecurityGroups(vei.Ctx, &ec2.DescribeSecurityGroupsInput{
+			GroupNames: []string{},
+			Filters: []ec2Types.Filter{
+				{
+					Name:   awsTools.String("vpc-id"),
+					Values: []string{vei.AWS.VpcID},
+				},
```

Comment:
```suggestion
				{
					Name:   awsTools.String("vpc-id"),
					Values: []string{vei.AWS.VpcID},
				},
				{
					Name:   awsTools.String("group-name"),
					Values: []string{"default"},
				},
```
We should be able to combine the filters here so that we can delete the filtering loop underneath this